dart/test/sentry_client_test.dart

```diff
@@ -551,3 +611,13 @@ SentryEvent beforeSendCallback(SentryEvent event, {dynamic hint}) {
 SentryEvent? eventProcessorDropEvent(SentryEvent event, {dynamic hint}) {
   return null;
 }
+
+class Fixture {
+  /// Test Fixture for tests with [SentryOptions.sendDefaultPii]
+  SentryClient getSut(bool sendDefaultPii, Transport transport) {
```

Comment:
It's approved already so feel free to ignore :)
but `Transport transport` could still be a global field of Fixture and accessed via `fixture.transport` as it does nothing but only mocked.

Reply:
Yeah, I wondered what would be the better way of doing that. Let's keep it as is for now.